### PR TITLE
Ecs guide

### PR DESCRIPTION
**Pull request by awssteph**
Added steps on how to use ECS solution

### CODE REVIEW

File: content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md [hunk 1]
```diff
@@ -255,6 +255,19 @@ The AccountCollector module is reusable and only needs to be added once but mult
 
 This module will enable you too automated report to show costs associated with ECS Tasks leveraging EC2 instances within a Cluster. Instructions on how to use this data can be found [here.](https://github.com/aws-samples/ecs-chargeback-cloudformation) This Data will be partitioned by year, month, day. 
 
+### Pre-Requisites  
+
+* Completion of  Well-Architected Lab: [100_1_aws_account_setup](https://wellarchitectedlabs.com/cost/100_labs/100_1_aws_account_setup/) or similar setup of the Cost and Usage Report (CUR) with resource Id enabled
+* A CUR file has been established for the existing Management/Payer account within the Billing Console.
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md:261]: inconsistent full stop at end of bullet point

File: content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md [hunk 2]
```diff
@@ -255,6 +255,19 @@ The AccountCollector module is reusable and only needs to be added once but mult
 
 This module will enable you too automated report to show costs associated with ECS Tasks leveraging EC2 instances within a Cluster. Instructions on how to use this data can be found [here.](https://github.com/aws-samples/ecs-chargeback-cloudformation) This Data will be partitioned by year, month, day. 
 
+### Pre-Requisites  
+
+* Completion of  Well-Architected Lab: [100_1_aws_account_setup](https://wellarchitectedlabs.com/cost/100_labs/100_1_aws_account_setup/) or similar setup of the Cost and Usage Report (CUR) with resource Id enabled
+* A CUR file has been established for the existing Management/Payer account within the Billing Console. 
+* The ECS Cluster leveraging EC2 instances for compute resides in a Linked Account connected to the Management Account through the "Consolidated Billing" option within the Billing Console. 
+* AWS generated tag is active in Cost Allocation Tags **aws:ecs:serviceName**  this will appear in the CUR as resource_tags_aws_ecs_service_Name
+* User-defined cost allocation tags *Name* is active
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md:264]: Cost Allocation Tags is capitalised on the line above.

File: content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md [hunk 3]
```diff
@@ -255,6 +255,19 @@ The AccountCollector module is reusable and only needs to be added once but mult
 
 This module will enable you too automated report to show costs associated with ECS Tasks leveraging EC2 instances within a Cluster. Instructions on how to use this data can be found [here.](https://github.com/aws-samples/ecs-chargeback-cloudformation) This Data will be partitioned by year, month, day. 
 
+### Pre-Requisites  
+
+* Completion of  Well-Architected Lab: [100_1_aws_account_setup](https://wellarchitectedlabs.com/cost/100_labs/100_1_aws_account_setup/) or similar setup of the Cost and Usage Report (CUR) with resource Id enabled
+* A CUR file has been established for the existing Management/Payer account within the Billing Console. 
+* The ECS Cluster leveraging EC2 instances for compute resides in a Linked Account connected to the Management Account through the "Consolidated Billing" option within the Billing Console.
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/3_Data_Collection_Modules.md:262]: Same inconsistent with full stop for bullet points

File: content/Cost/300_Labs/300_Optimization_Data_Collection/4_Utilize_Data.md [hunk 4]
```diff
@@ -230,4 +230,46 @@ If you wish to connect to your Cost and Usage report for snapshot costs please u
...
+
+Report to show costs associated with ECS Tasks leveraging EC2 instances within a Cluster
+{{%expand "Athena Configuration" %}}
+
+
+1. Navigate to the Athena service
+2. Select the "optimization data" database
+
+3. In **Saved Queries** find **"cluster_metadata_view"**" Change 'BU' to the tag you wish to do chargeback for
+4. Execute the query
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/4_Utilize_Data.md:243]: Should these be more like "Click the Execute query button"?

File: content/Cost/300_Labs/300_Optimization_Data_Collection/4_Utilize_Data.md [hunk 5]
```diff
@@ -230,4 +230,46 @@ If you wish to connect to your Cost and Usage report for snapshot costs please u
...
+8. Click the **Run** button
+
+Now your views are created you can run your report
+
+#### Manually execute billing report
+
+* In **Saved Queries** find **"ecs_chargeback_report"** 
+	-- Replace "bu_usage_view.month" value with the appropriate month desired for the report
+	-- For example, a value of '2' returns the charges for February 
+* Execute the query
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/4_Utilize_Data.md:259]: Should this be "Click the **Run** button" to keep it consistent with the section above?

Comment (awssteph) [reply content/Cost/300_Labs/300_Optimization_Data_Collection/4_Utilize_Data.md]: Done

**Review by awsdunc — APPROVED** [review]
LGTM

**Pull request merged**